src/pages/partners/payment-agent/_who-can-apply.js

```diff
@@ -45,27 +48,18 @@ const ImageWrapper = styled.img`
     height: 48px;
 `
 
+const LineSeparator = styled(Separator)`
+    height: 600px;
+    margin: 0 0;
```

Comment:
Does margin:auto accomplish the same thing ?

Reply:
oh yeah it does